plotting-app/pom.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
why we have 2 different `pom.xml` files?

Reply:
We have an outer module, and an inner module (the one used for the HiDDriver module). We will have more in the future.